_posts/0000-01-02-yvonmanzi.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
"The world breaks everyone, and afterwards, some are stronger in the broken places. Those that do not break, it kills."